benchmarks/benchmark-spring-data-jdbc/gradlew.bat

```diff
@@ -1,84 +1,84 @@
-@if "%DEBUG%" == "" @echo off

```

Comment:
This file always shows up as outgoing and I don't see changes so I just committed it here.